### ISSUE TITLE
No visual feedback when clicking on the Tx Share button 

### ISSUE THREAD

**Issue by katspaugh** [safe-global/safe-wallet-web#3283]
## Bug description

When clicking on the Share button in transaction details, there's no tooltip saying "Copied".

<img width="540" alt="Screenshot 2024-02-19 at 15 00 53" src="https://github.com/safe-global/safe-wallet-web/assets/381895/bbf88428-d37e-4487-a1f0-0fa909664433">

**Comment by daniel0ar** [comment]
Hi! I have seen where to add this to the button. Should I modify this to a copy button (check image)? Since it is only copying the address of the transaction url. Also, how do I sign the Safe Contributor License Agreement? I read that it is a requirement to contribute.
![image](https://github.com/safe-global/safe-wallet-web/assets/34529009/bf1b92fa-c859-47ac-bfec-aec000772d4a)

**Comment by daniel0ar** [comment]
Is this the outcome expected? If so please assign this to me and then I would send the PR. Also, is there a template for PR submission?
![image](https://github.com/safe-global/safe-wallet-web/assets/34529009/96038b3d-9d53-4fe4-9117-6c66266b79dd)
![image](https://github.com/safe-global/safe-wallet-web/assets/34529009/2903ce25-9a2c-4d7f-91ad-738345939712)

**Comment by katspaugh** [comment]
@daniel0ar exactly, thank you! Yes, there's a github template when you create a PR.

**Comment by MAGICXPLUS** [comment]
Y
